R/importance_sampling.R

```diff
@@ -0,0 +1,209 @@
+
+#' @rdname psis
+importance_sampling <- function(log_ratios, ...) UseMethod("importance_sampling")
+
+#' @noRd
+#' @description
+#' Currently implemented importance sampling methods
+implemented_is_methods <- function() c("PSIS", "TIS", "SIS")
```

Comment:
I would argue to use lower case version of the names as the function are called that way. In addition, I would actually argue to specify `is_method = c("psis", "tis", "sis")` in the function signature and then use the canonical `is_method <- match.arg(is_method)` for input validation, which nicely handles strange inputs and provides informative error messages.

Reply:
Alright! Sounds good. I'm used to use checkmate that solves this for you. =)